client/src/hooks/useKeyPress.ts

```diff
@@ -1,16 +1,47 @@
 import { useCallback, useEffect, useLayoutEffect, useRef } from 'react';
 
-const useKeyPress = (keys, callback, node = null) => {
+const useKeyPress = (
+    keys,
+    callback,
+    options = { useShift: false, useCtrl: true },
+    node = null
+) => {
     const callbackRef = useRef(callback);
     useLayoutEffect(() => {
         callbackRef.current = callback;
     });
 
-    const isShortcut = (event) => keys.some((key) => event.key === key);
+    const isShortcut = (event) => {
+        const ctrlOrCmdPressed = event.ctrlKey || event.metaKey;
+        const shiftPressed = event.shiftKey;
+
+        return keys.some((key) => {
+            const keyMatch = event.key.toLowerCase() === key;
+
+            if (keyMatch) {
+                if (
+                    options.useShift &&
+                    options.useCtrl &&
+                    ctrlOrCmdPressed &&
+                    shiftPressed
+                ) {
+                    return event.ctrlKey && event.shiftKey;
+                } else if (options.useCtrl && ctrlOrCmdPressed) {
+                    return event.ctrlKey;
+                } else if (options.useShift && shiftPressed) {
+                    return event.shiftKey;
+                } else if (!options.useShift && !options.useCtrl) {
+                    return keyMatch;
+                }
+            }
+
+            return false;
+        });
```

Comment:
what does this do?

and same as the previous comment, all changes that have to do with commas and semicolons alone should removed

Reply:
Delete it, it's not needed or used